src/main.rs

```diff
@@ -417,6 +418,22 @@ async fn check_data_folder() {
     }
 }
 
+fn check_push_config() {
+    if CONFIG.push_enabled()
+        && (CONFIG.push_installation_id() == String::new() || CONFIG.push_installation_key() == String::new())
+    {
+        error!(
+            "Misconfigured Push Notification service\n\
+            ########################################################################################\n\
+            # It looks like you enabled Push Notification feature, but you didn't configured the   #\n\
```

Comment:
```suggestion
            # It looks like you enabled Push Notification feature, but you didn't configure the    #\n\
```

Apart from the fact that this should go in `config.rs`, there's also a typo.